packages/runtime/runtime-utils/src/deltaManager.ts

```diff
@@ -31,3 +34,13 @@ export function toDeltaManagerErased(
 ): IDeltaManagerErased {
 	return deltaManager as unknown as IDeltaManagerErased;
 }
+
+/**
+ * Type guard to check if the given deltaManager is of type IDeltaManagerInternal.
+ * @internal
+ */
+export function isIDeltaManagerInternal(
```

Comment:
We defined this function elsewhere too. Can we deduplicate?

Reply:
Moved it to container-definitions